src/Views/FirmwareView.vala

```diff
@@ -327,7 +327,7 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
         }
 
         var message_dialog = new Granite.MessageDialog (
-            _("“%s” needs to manually be put in update mode").printf (device.get_name ()),
+            _("“%s” needs to be put in firmware-upgradable mode manually").printf (device.get_name ()),
```

Comment:
I'm not sure about `firmware-upgradable mode`. I think more likely user manuals will say `update mode`. Otherwise, I agree it sounds more natural to swap the order of these things in the sentence

Reply:
Differs between products. It is no good looking for the technically dubious "update mode" when actually wanting "Turn on AHCI support" for example.